### ISSUE TITLE
Comfyui- White screen at startup

### ISSUE THREAD

**Issue by Phantom-Lemech** [comfyanonymous/ComfyUI#5232]
### Expected Behavior

Launch the screen with a selection of workflows and opening schemes. It will run as usual!

### Actual Behavior

After updates, at startup, it enters the browser and a white screen and there is nothing to load, there is no interface at all, no manager, no workflow

### Steps to Reproduce

Запускаем Run run_nvidia_gpu the console and browser launch after a few seconds and then a white page - 
no program interface.

### Debug Logs

```powershell
D:\ComfyUI Test>.\python_embeded\python.exe -s ComfyUI\main.py --windows-standalone-build
[START] Security scan
[DONE] Security scan
## ComfyUI-Manager: installing dependencies done.
** ComfyUI startup time: 2024-10-13 13:21:19.921052
** Platform: Windows
** Python version: 3.11.9 (tags/v3.11.9:de54cf5, Apr  2 2024, 10:12:12) [MSC v.1938 64 bit (AMD64)]
** Python executable: D:\ComfyUI Test\python_embeded\python.exe
** ComfyUI Path: D:\ComfyUI Test\ComfyUI
** Log path: D:\ComfyUI Test\comfyui.log

Prestartup times for custom nodes:
   0.7 seconds: D:\ComfyUI Test\ComfyUI\custom_nodes\ComfyUI-Manager

Total VRAM 12288 MB, total RAM 65377 MB
pytorch version: 2.4.1+cu124
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 3060 : cudaMallocAsync
Using pytorch cross attention
[Prompt Server] web root: D:\ComfyUI Test\ComfyUI\web
D:\ComfyUI Test\python_embeded\Lib\site-packages\kornia\feature\lightglue.py:44: FutureWarning: `torch.cuda.amp.custom_fwd(args...)` is deprecated. Please use `torch.amp.custom_fwd(args..., device_type='cuda')` instead.
  @torch.cuda.amp.custom_fwd(cast_inputs=torch.float32)
### Loading: ComfyUI-Manager (V2.51.7)
### ComfyUI Revision: 2753 [5f9d5a24] *DETACHED | Released on '2024-10-09'

Import times for custom nodes:
   0.0 seconds: D:\ComfyUI Test\ComfyUI\custom_nodes\websocket_image_save.py
   0.2 seconds: D:\ComfyUI Test\ComfyUI\custom_nodes\ComfyUI-Manager

Starting server

To see the GUI go to: http://127.0.0.1:8188
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/model-list.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/alter-list.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/github-stats.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
```

### Other

There seems to be no error as such in the console, but still a white screen, and that happens in the latest version. That is, after the update. In  ComfyUI version 0.0.8 - everything works fine - it loads more in the console.

**Comment by ltdrdata** [comment]
It has been reported that some Chrome extensions and similar add-ons are blocking the loading of ComfyUI.
Try disabling extensions such as ad blockers and test again.
Also try running it in a different browser.

**Comment by Phantom-Lemech** [comment]
SOLVED - Update GIT and Python 3.13 at 07.10.2024
